### PR TITLE
Use snap UUID string instead of time format string.

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

File: snapshot/pkg/volume/gluster/snapshot.go [hunk 1]
```diff
@@ -60,9 +60,12 @@ func (h *glusterfsPlugin) SnapshotCreate(pv *v1.PersistentVolume, tags *map[stri
 	}
 
 	volumePath := spec.Glusterfs.Path
-	t := time.Now()
-	snapshotName := volumePath + "_" + t.Format(snapTimeFormat)
-
+	uuid := make([]byte, 16)
+	_, err := rand.Read(uuid)
```
Comment (tsmetana) [inline snapshot/pkg/volume/gluster/snapshot.go]: This looks rather wild... Wouldn't it be better to use "proper" UUID? We already have https://github.com/pborman/uuid among the dependencies anyway so I would definitely prefer using that.

Comment (humblec) [reply snapshot/pkg/volume/gluster/snapshot.go]: @tsmetana Sure.. Used `uuid` from referenced repo. Thanks.

**Comment by humblec** [comment]
@tsmetana Addressed the comment. Please review.

**Comment by humblec** [comment]
@tsmetana Can you please review this PR ?  @wongma7 the test failure is not related to this PR. 
```
[ERROR]	Failed to install: https://cloud.google.com/go Not Found

make[1]: *** [test-e2e] Error 1

make[1]: Leaving directory `/home/travis/gopath/src/github.com/kubernetes-incubator/external-storage/nfs'

make: *** [test-nfs] Error 2


```

Can you please take a look at it ?

**Comment by humblec** [comment]
@wongma7 PTAL :)

**Pull request merged**